web/src/components/core/Section.jsx

```diff
@@ -52,22 +52,26 @@ import { ValidationErrors } from "~/components/core";
  * @property {string} [path] - Path where the section links to.
  *  when user clicks on the title, used for opening a dialog.
  * @property {boolean} [loading] - Whether the section is busy loading its content or not.
+ * @property {string} [className] - Class name for section html tag.
  * @property {import("~/client/mixins").ValidationError[]} [props.errors] - Validation errors to be shown before the title.
  * @property {React.ReactElement} [children] - The section content.
  * @property {string} [aria-label] - aria-label attribute, required if title if not given
  *
  * @param { SectionProps } props
+ *
+ * @type {React.FC<SectionProps>}
  */
-export default function Section({
+const Section = forwardRef(function({
```

Comment:
I have doubts if we still needing the forwardRef in the Section component. Probably not by now. If so, do you mind dropping it until really needed?